src/syscheckd/src/db/src/db.cpp

```diff
@@ -100,6 +100,11 @@ DBSYNC_HANDLE DB::DBSyncHandle()
     return FIMDB::instance().DBSyncHandle();
 }
 
+void DB::registerRSync()
```

Comment:
Why are we exposing this? In the RunIntegrity the registration is already taking place. We are giving the caller responsibility for something that he can do implicitly.

FIMDB::runIntegrity()

Reply:
Yes, I didn't mind in that ! I have to add a mock to this call !